Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM python:3.8
+
+ENV MODELS_DIR=models
+ENV IMG_DIR=images
+ENV DEBIAN_FRONTEND=noninteractive
+
+WORKDIR /app
+
+COPY . .
+
+RUN apt-get update && \
+    apt-get install -y \
+        software-properties-common \
+        curl \
+        tar \
+        gzip \
+        git \
+        # python3 \
+        # python3-pip \
+        python3-opencv \
+        libgl1-mesa-glx \
+        tesseract-ocr-all \
+        pkg-config \
+        libtesseract-dev \
+        libleptonica-dev \
+        poppler-utils libpoppler-cpp-dev poppler-data \
+        cmake make g++ python3-dev wget default-jre libreoffice libreoffice-java-common \
+        && apt-get autoremove -fy \
+        && apt-get clean \
+        && apt-get autoclean -y
+
+RUN pip install --upgrade pip setuptools wheel
+
+RUN pip install -r .requirements.txt 
+
+RUN pip install mmcv-full==1.2.1 -f https://download.openmmlab.com/mmcv/dist/index.html
```

Comment:
With newest versions of pip it couldn't work, could you please download it directly and install from whl instead?

Reply:
Like this ```RUN mkdir mmcv && wget -P mmcv https://download.openmmlab.com/mmcv/dist/1.2.1/torch1.7.0/cpu/mmcv_full-1.2.1%2Btorch1.7.0%2Bcpu-cp38-cp38-manylinux1_x86_64.whl```